test/rpmtest/manifest.go

```diff
@@ -44,15 +45,28 @@ func PackagesFromRPMManifest(t *testing.T, r io.Reader) []*claircore.Package {
 			RepositoryHint: "key:" + rpm.GPG,
 			Module:         rpm.Module,
 		}
-		if s, ok := src[rpm.Source]; ok {
+
+		// Newer images produced from Konflux shove all the source information
+		// into the SourceName and omit the SourceNEVRA. Try both.
+		var source string
+		switch {
+		case rpm.SourceNEVRA != "":
+			source = rpm.SourceNEVRA
+		case rpm.SourceName != "":
+			source = rpm.SourceName
+		default:
+			continue
+		}
+
+		if s, ok := src[source]; ok {
 			p.Source = s
 		} else {
-			s := strings.TrimSuffix(rpm.Source, ".src")
+			s := strings.TrimSuffix(strings.TrimSuffix(source, ".rpm"), ".src")
```

Comment:
so it's either `.src` or `.src.rpm`?

Reply:
yeah, from the data the `SourceNEVRA` will have the `.rpm` , `SourceName` will have `.src.rpm`